F1WM/Controllers/NewsController.cs

```diff
@@ -49,7 +49,100 @@ public async Task<IActionResult> GetSingle(int id)
 			}
 		}
 
-		[HttpGet("important")]
+        [HttpGet("tag/{id}")]
+        [Produces("application/json", Type = typeof(NewsSummary))]
+        public async Task<IActionResult> GetByTag(int id)
```

Comment:
The method suggests that it gets news by a tag (string). GetByTagId(int tagId) would be more clear about the actual intent.